cartography/data/indexes.cypher

```diff
@@ -108,6 +108,7 @@ CREATE INDEX IF NOT EXISTS FOR (n:RDSSnapshot) ON (n.id);
 CREATE INDEX IF NOT EXISTS FOR (n:AWSESReservedInstance) ON (n.id);
 CREATE INDEX IF NOT EXISTS FOR (n:AWSSESIdentity) ON (n.id);
 CREATE INDEX IF NOT EXISTS FOR (n:GCPDNSZone) ON (n.id);
+CREATE INDEX IF NOT EXISTS FOR (n:GCPSQLDatabase) ON (n.id);
```

Comment:
let's move this next to `GCPSQLInstance`

Reply:
moved after GCPSQLInstance